app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,20 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+ .jumbotran {
+    position: relative;
```

Comment:
Properties should be ordered background-color, background-size, border, border-radius, height, overflow, padding, position, width